PixelHunter1995/SceneLib/Scene.cs

```diff
@@ -15,6 +15,7 @@ class Scene : ILoadContent
         private List<IUpdateable> Updateables;
         private List<ILoadContent> Loadables;
         private IDictionary<string, Portal> Portals;
+        public readonly string SongName;
```

Comment:
Should it be called "DefaultSongName" I assume that sometimes the song will change in the same scene?